designsafe/apps/api/notifications/migrations/0004_auto_20201019_1920.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2020-10-19 19:20
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Where do these migrations come from?

Reply:
I think they came just from running `./manage.py makemigrations`. Might have been from the switch to python3 wrt `bytes` string changes